command/certificate/needsRenewal.go

```diff
@@ -185,12 +210,26 @@ func needsRenewalAction(ctx *cli.Context) error {
 			percentUsed := (1 - remainingValidity.Minutes()/totalValidity.Minutes()) * 100
 
 			if int(percentUsed) >= percentThreshold {
-				return nil
+				return isVerboseExit(true, isVerbose)
 			}
 		} else if duration >= remainingValidity {
-			return nil
+			return isVerboseExit(true, isVerbose)
+		}
+
+		if !bundle {
+			break
 		}
 	}
 
+	return isVerboseExit(false, isVerbose)
+}
+
+func isVerboseExit(needsRenewal, isVerbose bool) error {
+	if isVerbose {
+		fmt.Println(needsRenewal)
+	}
+	if needsRenewal {
+		return nil
+	}
```

Comment:
I didn't know that when the certificate doesn't need renewal shows the following:

```
$ step certificate needs-renewal acme.crt
certificate does not need renewal
```

So perhaps the verbose option is not necessary.